pkg/api/handlers/compat/images.go

```diff
@@ -386,6 +388,12 @@ func LoadImages(w http.ResponseWriter, r *http.Request) {
 		utils.Error(w, "Something went wrong.", http.StatusInternalServerError, errors.Wrap(err, "failed to create tempfile"))
 		return
 	}
+	defer func() {
+		err := os.Remove(f.Name())
+		if err != nil {
+			logrus.Warnf("Failed to remove temporary file: %s.", f.Name())
```

Comment:
I wonder if this should be Warnf or Debugf....

Reply:
It should be a Errorf, and it should print out the error.  BTW when you print out the Error, it will include the f.Name(), so dont add it.

logrus.Errorf("Failed to remove temporary file: %v". err)

Reply:
@rhatdan, Good call.  My noggin was thinking last night that we'd be printing an error with every file we attempted to remove.  It will only print on ones we're trying to remove and fail to do so.